code/jack/python/lab07.py

```diff
@@ -0,0 +1,51 @@
+import math
```

Comment:
[Lab 7: Compute Automated Readability Index](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/07%20ARI.md)

* Needs Work
    * Missing sample file.

* Sample output:
```
PS C:\Users\Bruce\Programming\class_swordfish\code\jack\python> python .\lab07.py
Traceback (most recent call last):
  File "C:\Users\Bruce\Programming\class_swordfish\code\jack\python\lab07.py", line 6, in <module>
    with open(file_name, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'gettysburg_address.txt'
PS C:\Users\Bruce\Programming\class_swordfish\code\jack\python>
```

Reply:
Sample file for lab 7 added